pkg/oidcprovider/client.go

```diff
@@ -68,7 +69,10 @@ func (c client) IDTokenUserinfoClaimsAssertion() bool {
 // IsScopeAllowed implements op.Client.
 func (c client) IsScopeAllowed(scope string) bool {
 	slog.Warn("oidcprovider.client.IsScopeAllowed", "scope", scope)
-	panic("unimplemented")
+	if strings.HasPrefix(scope, "pls_") {
```

Comment:
Do we want to enforce that it must have something besides literally `pls_`? e.g. `pls_a` ok but `pls_` not